source/Grid/Grid.test.js

```diff
@@ -478,17 +478,28 @@ describe('Grid', () => {
   describe(':scrollLeft and :scrollTop properties', () => {
     it('should render correctly when an initial :scrollLeft and :scrollTop properties are specified', () => {
       let columnStartIndex, columnStopIndex, rowStartIndex, rowStopIndex
-      render(getMarkup({
+      render(getMarkup())
+      const rendered = findDOMNode(render(getMarkup({
         onSectionRendered: params => ({ columnStartIndex, columnStopIndex, rowStartIndex, rowStopIndex } = params),
         scrollLeft: 250,
         scrollTop: 100
-      }))
+      })))
+      expect(rendered.scrollTop).toEqual(100)
```

Comment:
Hm. This test doesn't pass for me (or [CI](https://circleci.com/gh/bvaughn/react-virtualized/797)).